python-flask/app.py

```diff
@@ -105,8 +105,29 @@ def unauthorized_response(callback):
     }), 401
 
 
+@app.route('/upload', methods=['POST'])
+def upload_file():
+    if request.method == 'POST':
+        files = request.files.to_dict()
+        for f in files:
```

Comment:
Just call it what it is, which is file in this case. It's just a personal style, not too big of a deal.